pkg/k8sutil/certs.go

```diff
@@ -294,6 +270,8 @@ func (k *K8sutil) CreateCertsSecret(namespace, clusterName, certsDir string) err
 			"node-key.pem":      nodeKey,
 			"kibana-key.pem":    kibanaKey,
 			"kibana.pem":        kibana,
+			"cerebro-key.pem":   cerebroKey,
```

Comment:
Are these certs that are generated used? Would be nice to serve cerebro over tls via these certs.